jbi/services/bugzilla.py

```diff
@@ -120,7 +138,28 @@ def get_client():
 def check_health() -> ServiceHealth:
     """Check health for Bugzilla Service"""
     client = get_client()
-    health: ServiceHealth = {"up": client.logged_in}
+    logged_in = client.logged_in
+
+    # Check that all JBI webhooks are enabled in Bugzilla,
+    # and report disabled ones.
+    all_webhooks_enabled = False
+    if logged_in:
+        webhooks = client.list_webhooks()
+        jbi_webhooks = [wh for wh in webhooks if "/bugzilla_webhook" in wh.url]
+        # See mozilla/jira-bugzilla-integration#21 whether it should
+        # be a single webhook with `product == "Any"`.
+        all_webhooks_enabled = len(jbi_webhooks) > 0
+        for webhook in jbi_webhooks:
+            if not webhook.enabled:
+                all_webhooks_enabled = False
+                logger.error(
+                    "Webhook %s is disabled (%s errors)", webhook.name, webhook.errors
+                )
+
+    health: ServiceHealth = {
+        "up": logged_in,
+        "all_webhooks_enabled": all_webhooks_enabled,
```

Comment:
If this is unhealthy; how do we notify about the error? _(sentry? logs? other?)_

Reply:
I think "errors" can be >0 for a substantial amount of time before the queue is "disabled" but the amount of errors would keep increasing.

I think both "disabled" and "errors" would be worthwhile to track for healthiness

Reply:
> If this is unhealthy; how do we notify about the error? (sentry? logs? other?)

In this case the `logger.error()` would send a Sentry error.

> I think both "disabled" and "errors" would be worthwhile to track for healthiness

Should we emit a warning when errors are `>0` then?

Reply:
> In this case the logger.error() would send a Sentry error.
👍 

> Should we emit a warning when errors are >0 then?
Yeah, a threshold of 0 sounds good.